src/libraries/System.Reflection.Metadata/src/System/Reflection/PortableExecutable/DebugDirectory/DebugDirectoryBuilder.cs

```diff
@@ -97,7 +97,7 @@ public void AddEntry<TData>(DebugDirectoryEntryType type, uint version, uint sta
         /// <exception cref="ArgumentException"><paramref name="pdbPath"/> contains NUL character.</exception>
         /// <exception cref="ArgumentOutOfRangeException"><paramref name="age"/> is less than 1.</exception>
         /// <exception cref="ArgumentOutOfRangeException"><paramref name="portablePdbVersion"/> is smaller than 0x0100.</exception>
-        internal void AddCodeViewEntry(
+        public void AddCodeViewEntry(
```

Comment:
Do we already have sufficient tests for this?

@carlossanlop how come it didn't get auto-labelled as new API needing docs?

Reply:
@tmat - thoughts on testing?